docs/proof-of-stake/ticket-splitting.md

```diff
@@ -0,0 +1,112 @@
+
+    - [https://stake.decredbrasil.com](https://stake.decredbrasil.com)
+    - [https://decredvoting.com](https://stake.decredbrasil.com)
+
+1. Login to your VSP and navigate to the `Settings` tab. Copy the API token.
+
+    ![VSP Settings page](../img/ticket_splitting_settings.png)
+
+1. Open your Decrediton wallet. Navigate to the `Tickets` tab, select your VSP from the Stakepool dropdown menu, and paste in your VSP API key. If you already have another VSP configured, add the new VSP as you would any other VSP. 
+
+    ![Ticket tab Decrediton](../img/ticket_splitting_decrediton.png)
+
+1. Enter your wallet passphrase. You should now see your configured stakepool.
+
+    ![Configured Decrediton Stakepool](../img/ticket_splitting_config_stakepool.png)
+
+1. Download the Split Ticket Matcher software for your OS.
+
+    - Split Ticket Matcher [downloads](https://github.com/matheusd/dcr-split-ticket-matcher/releases/tag/v0.7.0).
+    - Split Ticket Matcher instructions on [building from source](https://github.com/matheusd/dcr-split-ticket-matcher/blob/master/docs/building.md).
+
+1. Run the `splitticketbuyergui` executable. This will open the Split Ticket Buyer GUI (shown below). 
+
+    ![Split Ticket Buyer config menu](../img/ticket_splitting_gui.png)
+
+
+1. Click `Config -> Load from decrediton` and select your wallet name and VSP from the drop-down menus. 
+
+    ![Split Ticket Buyer config menu](../img/split_ticket_buyer_config_menut.png)
+
+1. Enter the amount of DCR you wish to put into your split ticket. Currently, only integer numbers are supported in the GUI version of Split Ticket Buyer. In the CLI version you can purchase arbitrary amounts. The minimum amount of DCR you can contribute is 5 DCR. 
+
+    ![Split Ticket Buyer config menu](../img/split_ticket_buyer_amount.png)
+
+1. Pick a session to join. Public session names can typically be found on the websites of VSPs that support ticket splitting, or in the ticket splitting chat channels (#ticket_splitting channel on Decred's Slack group or in the Telegram group located [here](https://t.me/dcrtktsplit)). 
+
+1. Enter the name of the ticket buying session you want to join and your wallet passphrase. Then click `Participate` to join the session. In the screenshot below, we're joining a public session, `decredvoting1`. You can also create private sessions with other participants. All that is required to create a private session is for all participants to use the same session name. 
+
+    ![Split Ticket Buyer config menu](../img/ticket_splitting_enter_session.png)
+
+    After successfully joining a session, you will see the number of other participants and the total amount of DCR currently in the pool.  
+
+    ![Split Ticket Buyer config menu](../img/ticket_splitting_joined_session.png)
+
+    You are now in the ticket splitting pool! Until the ticket price is reached, no DCR is spent. Participants are free to change the amount pledged, change wallets, or leave the pool altogether. 
```

Comment:
Actually, you can't change the amount before stopping participation, then restarting (changing the slider after you clicked on "Participate" doesn't really do anything).

Reply:
Reworded to:

> You are now in the ticket splitting pool! Until the ticket price is reached, no DCR is spent. Participants are free to the leave pool at any time. They can also re-enter the pool using a different wallet or pledging a different amount of DCR.

Reply:
Ok on the reword, but seems you haven't pushed it yet.

Reply:
Pushed. Have been wondering about the etiquette around that. Sometimes I push suggested changes. Sometimes I don't, with the reasoning that I don't want to "spam" the git history with too many commits. But actually, from a security perspective you have to push I suppose.

Reply:
This is really going to depend on each project, but in general you shouldn't worry about "spam" in git history.

Before actually merging the changes from a PR into master usually all commits on it will be merged into a single final commit, squashing all changes and removing the "noise" commits. Pushing individual commits into the PR itself is preferable in this case, since reviewers can see the individual changes.

Reply:
Right. I do that on code commits to other repos, but here on docs, it's a bit looser. Some are git noobs, and I guess the norm anyway is that a clean commit tree isn't as important with docs.

Reply:
Thanks for your quick help reviewing!